packages/components/src/components/carousel/carousel.tsx

```diff
@@ -65,6 +67,9 @@ export class Carousel {
   /** Defines the description used in the carousel. */
   @Prop() public description?: string;
 
+  /** Alignment of heading and description */
+  @Prop() public alignHeader?: HeaderAlign = 'left';
```

Comment:
Not sure about the prop name and also the type. In other components we just call this `align`. 
Also it could be confused with heading. And the prop name does not match the type: `align header <=> header align`?
For the type I'd suggest a `Carousel` prefix, so it would become `CarouselHeaderAlign`.

See for a complete list of all props https://designsystem.porsche.com/issue/v3-alpha/component-meta

Reply:
@denyo Regarding the name `alignHeader`: I think it's logical to call it `alignHeader`, because otherwise the user might think that the whole carousel will be aligned, not only header, and it might lead to confusion. And I clarified the name with Stephan & Marcel, and they approved it. We have properties with similar names, like `alignContent`, `alignItems`, `alignLabel`.

Reply:
Renamed type `HeaderAlign` -> `CarouselAlignHeader`